dojo/tools/semgrep/parser.py

```diff
@@ -67,7 +67,7 @@ def get_findings(self, filename, test):
 
     def convert_severity(self, val):
         if "WARNING" == val.upper():
-            return "Low"
+            return "Medium"
```

Comment:
It looks weird to me, to map a Warning to Medium instead of Low.

Reply:
@StefanFl It's what sarif does. Ultimately we have the problem of mapping external severity levels that only cover 3 possibilities into the 5 in DefectDojo. The goal of the PR is to at least maintain consistency between the mapping levels, because for example we can ingest semgrep using both the semgrep and sarif parsers and end up with different severity levels...